vue/components/ui/atoms/button-color/button-color.vue

```diff
@@ -124,6 +124,10 @@
     border-color: #2d2d2d;
 }
 
+.button-color.button-color.button-color-medium {

```

Comment:
This doesn't seem right, shouldn't it be `.button-color.button-color-medium`

Reply:
Also this needs to go before before line 77, before `.button-color.button-color-small`